packages/client/src/screens/auth/login/index.tsx

```diff
@@ -82,13 +84,7 @@ export const LoginScreen: React.FC = () => {
                 error={touched.password ? errors.password : undefined}
               />
               <FlexContainer marginBottom='2rem'>
-                <Button
-                  disabled={loging}
-                  type='submit'
-                  primary
-                  fit
-                  className='fit'
-                >
+                <Button disabled={loging} type='submit' primary fit>
```

Comment:
Change `loging` to `loggingIn`, loging does not stand for logging in, they're two different verbs, therefor the name is confusing to whoever reads this.